internal/data/spicedb_test.go

```diff
@@ -180,6 +183,80 @@ func TestSecondCreateRelationshipSucceedsWithTouchTrue(t *testing.T) {
 	assert.True(t, exists)
 }
 
+type MockgRPCClientStream struct {
+	mock.Mock
+}
+
+func (m *MockgRPCClientStream) SetHeader(md metadata.MD) error {
+	panic("implement me")
+}
+
+func (m *MockgRPCClientStream) SendHeader(md metadata.MD) error {
+	panic("implement me")
+}
+
+func (m *MockgRPCClientStream) SetTrailer(md metadata.MD) {
+	panic("implement me")
+}
+
+func (m *MockgRPCClientStream) Context() context.Context {
+	panic("implement me")
+}
+
+func (m *MockgRPCClientStream) SendMsg(_ any) error {
+	panic("implement me")
+}
+
+func (m *MockgRPCClientStream) RecvMsg(_ any) error {
+	panic("implement me")
+}
+
+func (m *MockgRPCClientStream) Recv() (*apiV1beta1.ImportBulkTuplesRequest, error) {
+	args := m.Called()
+	if req, ok := args.Get(0).(*apiV1beta1.ImportBulkTuplesRequest); ok {
+		return req, args.Error(1)
+	}
+	return nil, args.Error(1)
+}
+
+// SendAndClose simulates sending a response and closing the stream
+func (m *MockgRPCClientStream) SendAndClose(resp *apiV1beta1.ImportBulkTuplesResponse) error {
+	args := m.Called(resp)
+	return args.Error(0)
+}
+
+func (m *MockgRPCClientStream) CloseAndRecv() (*apiV1beta1.ImportBulkTuplesResponse, error) {
+	args := m.Called()
+	if res, ok := args.Get(0).(*apiV1beta1.ImportBulkTuplesResponse); ok {
+		return res, args.Error(1)
+	}
+	return nil, args.Error(1)
+}
+
+func TestImportBulkTuples(t *testing.T) {
+	rels := []*apiV1beta1.Relationship{
+		createRelationship("rbac", "group", "bob_club", "t_member", "rbac", "user", "bob5", ""),
```

Comment:
You shouldn't need to prefix member with "t_", it should be automatically prefixed without you needing to manually write it. There exists a helper function `addRelationPrefix` that can be used during the creation stage to add the prefix.

Reply:
fixed